src/lib/wpcli-versions.ts

```diff
@@ -0,0 +1,66 @@
+import fs from 'fs-extra';
+import semver from 'semver';
+import { downloadWpCli } from '../../vendor/wp-now/src/download';
+import { executeWPCli } from '../../vendor/wp-now/src/execute-wp-cli';
+import getWpCliPath from '../../vendor/wp-now/src/get-wp-cli-path';
+
+export async function updateLatestWPCliVersion() {
```

Comment:
```suggestion
export async function updateLatestWpCliVersion() {
```
Can we make the `WpCli` name consistent across function and variable names?